docs/running-tests.md

```diff
@@ -4,4 +4,88 @@ title: Running tests
 nav_order: 4
 ---
 
-Will do this via DEVEXP-525.
+Tests written using marklogic-unit-test can be run in several ways, each of which is described below. 
+
+## Using the web application
+
+Once you have deployed your application along with the marklogic-unit-test modules, as described in the 
+[Getting started guide](getting-started.md), you will be able to access a custom endpoint via any MarkLogic HTTP 
+App Server associated with the modules database containing your application modules. The path of that endpoint is 
+`/test/default.xqy`. For example, since the App Server in the [Getting started guide](getting-started.md) listens on 
+port 8024, the endpoint would be accessible at <http://localhost:8024/test/default.xqy>.
+
+The web application at this custom endpoint provides a functional, albeit antiquated-looking, interface for running
+one or more tests and/or suites at a time. Use the checkboxes under "Run" and "Run All Tests" to select which tests
+and suites you would like to run. Click the "Run Tests" button in the upper right-hand corner to run the selected
+tests. Any failed test will display a message capturing the assertion failure. 
+
+## Using Gradle 
+
+The [ml-gradle plugin for Gradle](https://github.com/marklogic/ml-gradle) provides an `mlUnitTest` task for running 
+all of your marklogic-unit-test tests. As noted in the [Getting started guide](getting-started.md), you will need to 
+include the following block at the top of your `build.gradle` file to ensure this task can run successfully:
+
+```
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+  dependencies {
+    classpath "com.marklogic:marklogic-unit-test-client:1.3.0"
+  }
+}
+```
+
+You can then run the task:
+
+    ./gradlew mlUnitTest
+
+The output of this task will capture how many tests passed and how many failed. A failure message will be displayed for
+each test that fails. 
+
+Please see 
+[the ml-gradle example project for marklogic-unit-test](https://github.com/marklogic/ml-gradle/tree/master/examples/unit-test-project)
+for more information on how tests are run, along with more information on how to configure the connection to a MarkLogic
+App Server.
+
+## Using JUnit5
```

Comment:
This really ought to have its own page, as it's a much bigger topic. This at least links to the README / example project, which is better than what we have today. But that README ought to be folded into the real docs here at some point.